index.html

```diff
@@ -546,6 +581,9 @@ <h3>Role Mapping Table</h3>
 				</tr>
 				<tr id="role-map-cell">
 					<th><a class="role-reference" href="#cell"><code>cell</code></a></th>
+					<td class="role-computed">
+						<p><code>cell</code></p>
+					</td>
```

Comment:
For these, should we specify something more like:

Use the computed role <p><code>cell</code></p> when the element is in the appropriate context, and the native host language role otherwise.

If we do this, do we need to define a term "context" maybe.

Reply:
Perhaps that could be in the computed role section (referencing required context), rather than in the table for each.